modules/installation-restricted-network-samples.adoc

```diff
@@ -34,7 +34,7 @@ endif::[]
 
 Setting the `samplesRegistry` field in the Sample Operator configuration file to link:https://registry.redhat.io[registry.redhat.io] is redundant because it is already directed to link:https://registry.redhat.io[registry.redhat.io] for everything but Jenkins images and imagestreams. It also breaks the installation payload for Jenkins imagestreams.
```

Comment:
"It also breaks the installation payload for Jenkins imagestreams." need removed. `samplesRegistry` can't override Jenkins imagestreams now.